packages/twenty-server/src/engine/core-modules/user-workspace/user-workspace.entity.ts

```diff
@@ -79,6 +88,10 @@ export class UserWorkspace {
   @Column({ nullable: false, default: SOURCE_LOCALE, type: 'varchar' })
   locale: keyof typeof APP_LOCALES;
 
+  @Field(() => PreferredCalendar, { nullable: false })
+  @Column({ type: 'varchar', default: PreferredCalendar.Gregorian })
```

Comment:
column should be non nullable too (which is fine for existing rows as long as we have a default)

Reply:
we should also use an enum on postgres side (we only register it in graphql here if I'm not mistaken)

Reply:
we should also use an enum on postgres side (we only register it in graphql here if I'm not mistaken)